--- xfce4-timer-plugin-0.6.1/configure.in~	2008-11-20 21:45:22.000000000 +0100
+++ xfce4-timer-plugin-0.6.1/configure.in	2011-07-06 10:49:47.560501986 +0200
@@ -32,7 +32,7 @@
 dnl required
 XDT_CHECK_PACKAGE([GTHREAD], [gthread-2.0], [2.4.0])
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [2.4.0])
-XDT_CHECK_PACKAGE([LIBXFCEGUI4], [libxfcegui4-1.0], [4.2.0])
+XDT_CHECK_PACKAGE([LIBXFCE4UI], [libxfce4ui-1], [4.2.0])
 XDT_CHECK_PACKAGE([LIBXFCE4PANEL], [libxfce4panel-1.0], [4.3.20])
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.3.0])
 
--- xfce4-timer-plugin-0.6.1/src/xfcetimer.c~	2008-11-20 21:45:21.000000000 +0100
+++ xfce4-timer-plugin-0.6.1/src/xfcetimer.c	2011-07-06 11:08:57.264517246 +0200
@@ -36,9 +36,8 @@
 #include <gtk/gtk.h>
 
 #include <libxfce4util/libxfce4util.h>
-#include <libxfcegui4/xfce_iconbutton.h>
-#include <libxfce4panel/xfce-panel-plugin.h>
-#include <libxfcegui4/dialogs.h>
+#include <libxfce4ui/libxfce4ui.h>
+#include <libxfce4panel/libxfce4panel.h>
 
 #include "xfcetimer.h"
 
--- xfce4-timer-plugin-0.6.1/src/Makefile.am~	2008-11-20 21:45:21.000000000 +0100
+++ xfce4-timer-plugin-0.6.1/src/Makefile.am	2011-07-06 11:13:21.227027425 +0200
@@ -18,11 +18,11 @@
 xfce4_timer_LDFLAGS =						\
 	@LIBXFCE4PANEL_LIBS@ \
 	@LIBXFCE4UTIL_LIBS@ \
-	@LIBXFCEGUI4_LIBS@
+	@LIBXFCE4UI_LIBS@
 		
 xfce4_timer_CFLAGS =						\
 	-I$(top_srcdir) \
-	@LIBXFCEGUI4_CFLAGS@ \
+	@LIBXFCE4UI_CFLAGS@ \
 	@LIBXFCE4UTIL_CFLAGS@ \
 	@LIBXFCE4PANEL_CFLAGS@ \
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"
--- xfce4-timer-plugin-0.6.4/src/xfcetimer.c.orig	2012-05-17 23:27:50.106931152 +0200
+++ xfce4-timer-plugin-0.6.4/src/xfcetimer.c	2012-05-17 23:32:20.133588295 +0200
@@ -1395,7 +1395,7 @@
 
   xfce_panel_plugin_block_menu (plugin);
 
-  dlg = gtk_dialog_new_with_buttons (_("Xfce 4 Timer Plugin"),
+  dlg = xfce_titled_dialog_new_with_buttons (_("Xfce 4 Timer Plugin"),
               GTK_WINDOW (gtk_widget_get_toplevel (GTK_WIDGET (plugin))),
               GTK_DIALOG_DESTROY_WITH_PARENT |
               GTK_DIALOG_NO_SEPARATOR,
@@ -1407,12 +1407,7 @@
 
   gtk_container_set_border_width (GTK_CONTAINER(dlg), BORDER);
 
-  header = xfce_create_header (NULL, _("Xfce4 Timer Options"));
-  gtk_widget_set_size_request (GTK_BIN (header)->child, 200, 32);
-  //gtk_container_set_border_width (GTK_CONTAINER (header), 6);
-  gtk_widget_show (header);
-  gtk_box_pack_start (GTK_BOX (GTK_DIALOG (dlg)->vbox), header,
-                      FALSE, TRUE, 0);
+  xfce_titled_dialog_set_subtitle (XFCE_TITLED_DIALOG(dlg), _("Xfce4 Timer Options"));
 
   gtk_box_pack_start (GTK_BOX (GTK_DIALOG (dlg)->vbox), vbox,
                       TRUE, TRUE, WIDGET_SPACING);
